### PR TITLE
Update HTTP metrics and views.

### PR DESCRIPTION
**Pull request by ramonza**
* Simplify and reduce requirements
* Bring into closer alignment with gRPC
* Specify units

### CODE REVIEW

File: http/stats.md [hunk 1]
```diff
@@ -6,24 +6,32 @@ All times (latencies) are measured in float64 fractions of microseconds.
 
 There is no special support for multi-part HTTP requests and responses. These are just treated as one request.
 
+## Units
+
+* Latencies are measures in float64 milliseconds, denoted "ms"
+* Sizes are measured in bytes, denoted "By"
```
Comment (HailongWen) [inline http/stats.md]: I think "By" might be confused with "by". How about using only "B" to denote "byte"?

Comment (ramonza) [reply http/stats.md]: These are from http://unitsofmeasure.org/ucum.html (also used by Stackdriver)

Comment (ramonza) [reply http/stats.md]: Added a note to this effect

Comment (HailongWen) [reply http/stats.md]: I see, thanks!

File: http/stats.md [hunk 2]
```diff
@@ -1,53 +1,60 @@
...
-| status_code   | HTTP status code, or 0 if no response status line was received |
-| response_type | Response media type, if applicable                             |
-| request_type  | Request media type, if applicable                              |
-| version       | HTTP version in request                                        |
-| path          | URL path (not including query string)                          |
-| host          | Value of the request Host header                               |
+| Tag suffix  | Description                                                          |
+|-------------|----------------------------------------------------------------------|
+| http.method | HTTP method, capitalized (i.e. GET, POST, PUT, DELETE, etc.)         |
+| http.status | HTTP status code, or "error" if no response status line was received |
```
Comment (odeke-em) [inline http/stats.md]: Did we mean `http.status_code` here?

**Pull request merged**